src/drive/targets/mobile/InitAppMobile.jsx

```diff
@@ -23,11 +23,8 @@ import {
   startHeartBeat,
   stopHeartBeat
 } from 'drive/mobile/lib/tracker'
-import {
-  getLang,
-  initClient,
-  registerClientPlugins
-} from 'drive/mobile/lib/cozy-helper'
+import { getLang, initClient } from 'drive/mobile/lib/cozy-helper'
+import registerClientPlugins from 'drive/lib/registerClientPlugins'
```

Comment:
Could  registerClientPlugins be in initClient ?

Reply:
No because initClient is also used in the background process where we don't want the plugin to be registered. We also want to register the plugin on the web where we don't use initClient at all. All in all I think merging the 2 together would make the code harder to understand and maintain.

Reply:
>  All in all I think merging the 2 together would make the code harder to understand and maintain.

Why ? I think that having the same initialization code makes it easier to understand. Typically I think what is important to understand are the differences between mobile and web. Here since this is done in two different files, it's hard to compare, don't you think ? And why is the web version not using initClient ? Doesn't it have to initialize a client ?

Reply:
The client initialization for web and mobile are wildly different (different links, token handling, etc). We could have a single function that inits cozy client for all contexts, but it would have a bunch of `if mobile / if background / etc` in it, which I don't think is preferable.

The other reason why I don't think we should make this change is that we need to import a lot of files to init cozy-client on mobile, most of which are not relevant at all on the web and should ideally not be in the bundle. The file structure in drive is not ideal, and it's easy to end up including way too many things because of chained imports. This is what caused the bug I'm addressing here in the first place — accidentally including cozy-helper in the intents bundle broke it.

Reply:
OK, fine by me.